packages/push/examples/complex_regression/main.rs

```diff
@@ -110,7 +142,7 @@ fn main() -> Result<()> {
 
     let population = gene_generator
         .to_collection_generator(args.max_initial_instructions)
-        .with_scorer(&scorer)
+        .with_scorer_fn(&scorer)
```

Comment:
Same thing as with FnScorer also applies here - I would use `.with_scorer(&scorer)` and wrap `scorer` with an `FnScorer` at initialization

Reply:
We'll have the same problem with `.wrap()` farther down in the file.